fairscale/nn/pipe/pipe.py

```diff
@@ -255,6 +256,13 @@ def __init__(
     ) -> None:
         super().__init__()
 
+        if torch.__version__.split(".")[:2] >= ["1", "8"]:
+            warnings.warn(
+                "fairscale.nn.Pipe has been upstreamed to PyTorch as torch.distributed.pipeline.sync.Pipe. "
```

Comment:
Do we need to mention "with minor API changes"? So, people do not expect their code should just work by adjusting import path.